ui/component/or-attribute-input/src/or-anomaly-config-input.ts

```diff
@@ -0,0 +1,358 @@
+                                  @or-mwc-input-changed="${(e: OrInputChangedEvent) => {
+                                      e.detail.value = types.filter((obj) => obj.label === e.detail.value).map((obj) => obj.value)[0]
+                                      this.updateProperty(e,"type");
+                                  }}"></or-mwc-input>
+                    ${DetectionMethodConfig(method)}
+                </div>
+                
+            `;
+        }
+        const DetectionMethodConfig = (method: AnomalyDetectionConfigurationUnion) =>{
+            switch (method.type){
+                case "change":
+                    return html`
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="number" .label="${i18next.t("anomalyDetection.deviationChange")}" .value="${method.deviation}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"deviation")}"></or-mwc-input>
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="text" .label="${i18next.t("anomalyDetection.minimumTimespan")}" .value="${method.timespan}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"timespan")}"></or-mwc-input>
+                    `;
+                case "global":
+                    return html`
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="number" .label="${i18next.t("anomalyDetection.deviationGlobal")}" .value="${method.deviation}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"deviation")}"></or-mwc-input>
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="text" .label="${i18next.t("anomalyDetection.minimumTimespan")}" .value="${method.timespan}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"timespan")}"></or-mwc-input>
+                    `;
+                case "forecast":
+                    return html`
+                        <or-mwc-input style="padding-top: 10px;" required="true" type="number" .label="${i18next.t("anomalyDetection.deviationGlobal")}" .value="${method.deviation}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.updateProperty(e,"deviation")}"></or-mwc-input>
+                    `;
+                default:
+                    return html`
+                        
+                    `;
+            }
+        }
+        window.setTimeout(() => doLoad(this.anomalyDetectionConfigObject as AnomalyDetectionConfigObject), 0);
+        this.requestUpdate();
+        return html`
+            <style>
+                .test {
+                    display: flex;
+                    flex-direction: column;
+                    width: 100%;
+                }
+                .item{
+                    display: flex;
+                    width: 100%;
+                }
+            </style>
```

Comment:
Style tags shouldn't be included in the render. You should use one of these solutions;
https://lit.dev/docs/v2/components/styles/